composer.json

```diff
@@ -43,7 +43,7 @@
         "php": "^7.1",
         "laminas/laminas-component-installer": "^2.1.2",
         "laminas/laminas-config-aggregator": "^1.2",
-        "laminas/laminas-diactoros": "^1.8.7 || ^2.2.2",
+        "laminas/laminas-diactoros": "^1.8.7 || ^2.3.0",
```

Comment:
For a new minor release, we can bump this to just `^2.3.0`.

Reply:
ok, implemented.